ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -935,6 +852,7 @@ func resourceIBMContainerVpcClusterUpdate(d *schema.ResourceData, meta interface
 
 	return resourceIBMContainerVpcClusterRead(d, meta)
 }
+
```

Comment:
sorry, cannot comment on the line. but the Read function may need some tweaks too.
it expect a workerpool with the name "default" to fill in the workerpool fields. not sure if these codes are needed at all...
`workerPool, err := csClient.WorkerPools().GetWorkerPool(clusterID, "default", targetEnv)`

Reply:
I left it there on purpose. While they should no affect on working, as a source of information may be handy.